applications/plugins/SceneCreator/SceneCreatorConfig.cmake.in

```diff
@@ -3,8 +3,11 @@
 @PACKAGE_INIT@
 
 find_package(SofaSimulation REQUIRED)
-find_package(SofaMisc REQUIRED)
-find_package(Eigen3 QUIET REQUIRED)
+find_package(Eigen3 REQUIRED)
+
+if(SCENECREATOR_HAVE_SOFASPARSESOLVER)
+	find_package(SofaSparseSolver REQUIRED)
+endif()
```

Comment:
SCENECREATOR_HAVE_SOFASPARSESOLVER has no value in the scope of this file.
You need to add `set(SCENECREATOR_HAVE_SOFASPARSESOLVER @SCENECREATOR_HAVE_SOFASPARSESOLVER@)` to transfer the value from the scope of SceneCreator/CMakeLists.txt at configure time.